controlplane/api/v1alpha1/condition_consts.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
We should talk about licence/copyright headers

Reply:
Yes ... we should also have a talk about my copy-pasta ... :disappointed: